### DISCUSSION TITLE
Need help please

### DISCUSSION THREAD

**Discussion by Lmb29860** [community/community#44945] [New to GitHub] [answered]
Someone keeps hitting every device I have changing the data to managed account. I believe it is a neighbor! Can ANYONE help me?! Peace

**Comment by jge162** [comment]
It sounds like your devices may have been hacked, and someone is gaining unauthorized access to them. The first step to take is to change all of your passwords, and make sure that they are strong and unique. You should also enable two-factor authentication on any accounts that support it.

**Comment by dhengGhuring** [comment]
Maybe your device was getting hacked by someone, you must enable two-factor auth to get more secure your account

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Jaded911 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
